conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: graphistry-dev
+  version: "0.25.2"
```

Comment:
The version seems hard-coded here, while it should be taken from the git tag -- am I misreading somehow?

Reply:
Maybe: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#templating-with-jinja

Reply:
Working on it, for some reason this env variable is not present

Reply:
Resolved by switching to different github action